src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -223,7 +225,11 @@
         set-title-bottom           (rn/use-callback
                                     (fn [_ y _ height]
                                       (reset! title-bottom-coord
-                                        (+ y height -100 (if platform/ios? (- top) top)))))
+                                        (+ y
+                                           height
+                                           -56
+                                           (when platform/ios?
+                                             (* top -2))))))
```

Comment:
- magic numbers: -56, -2

- What is the difference between y and height?

- Why do we have this condition for ios: `(when platform/ios? (* top -2))`

Reply:
> magic numbers: -56, -2

-56 should be a magic number, it is the given height for the header.

> What is the difference between y and height?

Please check the API: https://reactnative.dev/docs/direct-manipulation#measurecallback

> Why do we have this condition for ios: (when platform/ios? (* top -2))

On iOS the content of the scroll view is rendered out of the safe area, so we add a `:margin -top` to the container, Since we added that negative margin, the scroll amount starts at `-top` instead of `0`, so we need to substract another `-top` to properly report the scroll amount.

It may be easier to grasp if you run the branch and check the printed logs (printed by the leftovers I previously mentioned).

Reply:
> -56 should be a magic number, it is the given height for the header.

You can define it as `(def header-height 56)`, it wasn't clear to me that it is the header height

> Please check the API: https://reactnative.dev/docs/direct-manipulation#measurecallback

I understand the styling difference :) I meant make it more clear what `y` and `height` are referring to? Y of what and height of what?

> the scroll amount starts at -top instead of 0, so we need to substract another -top to properly report the scroll amount.

👍